### DISCUSSION TITLE
Update eclipse-zenoh from 0.5.0-b9 to 0.6.0-b1

### DISCUSSION THREAD

**Discussion by paloma-jim** [eclipse-zenoh/roadmap#78] [Q&A]
Hello,

I have a docker container with eclipse-zenoh==0.5.0-b9. Due to some changes in the requirements of the project, i need to update it to 0.6.0-b1. However, when I run:

`pip3 install eclipse-zenoh==0.6.0-b1`

It gives me the following error. It is like it was not concious that there is a new version of Zenoh. 
`ERROR: Could not find a version that satisfies the requirement eclipse-zenoh==0.6.0-b1 (from versions: 0.4.2.dev3, 0.4.2.dev5, 0.4.2rc1, 0.5.0b4, 0.5.0b5, 0.5.0b7, 0.5.0b8, 0.5.0b9)
ERROR: No matching distribution found for eclipse-zenoh==0.6.0-b1`

Do you know what i am doing wrong and how can I fix this issue?

Thank you

**Comment by gabrik** [comment]
Hi @paloma-jim can you please provide us with your python version? Run`pyhton3 --version` and paste the result here.
My uninformed guess is that the python version you use is not supported by Zenoh 0.6.0-b1
